.gitignore

```diff
@@ -18,11 +18,44 @@
 /compatibility/head_sdk/
 .bazelrc.local
 .bazelproject
-.ijwb
 
 # node.js
 node_modules/
 
+
+### Operating Systems ###
+
 # macOS
 .DS_Store
 ._.DS_Store
+
+
+### Editors ###
+
+# Eclipse
+.classpath
+.factorypath
+.project
+.settings
+
+# Emacs
+\#*\#
+.\#*\#
+
+# Ensime
+.ensime
+.ensime_cache
+
+# IntelliJ IDEA
+.idea
+.ijwb # Bazel
+*.iml
+*.ipr
+*.iws
+out
+
+# Vim
+*~
```

Comment:
This is also emacs!

Reply:
Haha! I need to add the Vim swap files back in, too.

Reply:
Duplicated it for Emacs; if we delete one this should still work.